apps/wildfires/src/app/shared/components/usefulLinks/UsefulLinks.tsx

```diff
@@ -0,0 +1,39 @@
+import { BenefitsCalIcon, DeoIcon, EddIcon } from '@monorepo/react/icons';
+import Fema from '../../../../assets/images/FEMA-Logo.png';
+import UsefulLinksCard from './UsefulLinksCard';
+
+export default function UsefulLinks() {
+  return (
+    <div className="bg-brand-angel-blue py-12 w-full flex flex-col items-center justify-center">
+      <h2 className="text-2xl md:text-[40px] md:leading-[94.5px] font-bold mb-6">
+        Useful Links/Info
+      </h2>
+      <div className="flex flex-col md:flex-row items-stretch justify-center gap-1">
```

Comment:
1. the logos should be vertically and horizontally centered within the component
2. on desktop, the components aren't the same width. the parents are as wide as their narrowest child

<img width="593" alt="image" src="https://github.com/user-attachments/assets/f3dfd56b-cf9c-4412-9675-7cce1c2a6eb5" />